gef.py

```diff
@@ -10241,6 +10261,32 @@ def __parse_gdb_info_sections(self) -> Generator[Section, None, None]:
                 continue
         return
 
+    def __parse_info_mem(self) -> Generator[Section, None, None]:
+        """Get the memory mapping from GDB's command `monitor info mem`"""
+        stream = StringIO(gdb.execute("monitor info mem", to_string=True))
+        for line in stream:
```

Comment:
```suggestion
        for line in StringIO(gdb.execute("monitor info mem", to_string=True)):
```

Reply:
Note: I know I did the same in the function above, but might as well improve what we can now 😊